modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -2075,6 +2075,9 @@ public void testGetObjectEntryRelatedObjectEntriesWithAccountEntryRestricted()
 
 		_objectRelationshipLocalService.deleteObjectRelationship(
 			objectRelationship2.getObjectRelationshipId());
+
+		objectDefinitionLocalService.deleteObjectDefinition(
+			childObjectDefinition);
```

Comment:
In this case, we need only delete the `childObjectDefinition` and the `objectRelationship1` and `objectRelationship2` will be delete in the deleting objectDefinition process.

Reply:
You're right. Just updated the PR